python/paddle/jit/dy2static/partial_program.py

```diff
@@ -1119,5 +1155,8 @@ def add_build_strategy_for(
         if hasattr(compiled_program._program, 'lr_sheduler'):
             builded_program.lr_sheduler = compiled_program._program.lr_sheduler
     else:
-        builded_program = program
+        # can't just create a new program, we need copy the vardesc.
+        builded_program = paddle.static.Program()
```

Comment:
why this?

Reply:
fix the bug when the program only contain a var.
```python
@paddle.jit.to_static
def f(x):
    return x
```

Reply:
ok